content/docs/reference/design-docs/subcategory/index.en.md

```diff
@@ -0,0 +1,71 @@
+---
+title: "Sub category"
+linkTitle: "Sub category"
+weight: 60
+description: "Describes the sub-category model and its relation to train schedules and paced trains."
+---
+
+## Introduction
+
+Sub categories are created via a dedicated API. Each sub category has a name and a unique code, as well as visual properties like **color**, **hovered_color**, and **background_color**.
+Each sub category is associated with a parent main category.
+Both **TrainSchedules** and **PacedTrains** can specify either a **main_category** or a user-defined **sub_category_code**.
+
+### SubCategory model
+
+```yml
+name: "RER"
+code: "RER"
+main_category: "COMMUTER_TRAIN"
+color: "#FF0000"
+hovered_color: "#FFA500"
+background_color: "#FFD700"
+```
+
+### Train Schedule & Paced Train Category
+
+The category field in train schedules and paced trains can take either:
+
+```yml
+category:
+  main_category: "COMMUTER_TRAIN"
+```
+
+Or
+
+```yml
+category:
+  sub_category_code: "RER"
+```
+
+Only one of the two can be defined at a time, or both can be absent.
+
+If a referenced sub category is deleted, the affected train schedules or paced trains will automatically fall back to the sub category’s associated **main_category**.
```

Comment:
It's not clear from the documentation how the association between the sub category and its main category is done. We might need something to explain it?